nss-day-cohort-47/holiday-road-salt-lake-skaters
# Tl adding css

# Description

Adjusted some CSS to fix center containers. Played around with colors. Added a few details to the details container and made keys bold. Feel free to change color scheme!

Fixes # (issue)

## Type of change



# Testing Instructions
fetch and serve. test dropdowns and details to ensure text stays contained in appropriate containers.

# Checklist:

- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
